template/.npmignore.temp

```diff
@@ -12,11 +14,15 @@ docs/
 scripts/
 test/
 apiExamples/
+docTemplates/
 demo/**/*.css
 demo/**/*.scss
 demo/**/*.html
 
 dist/es5.js
+dist/auro-*.js
+!dist/auro-*__bundled.js
+dist/style-css.js
```

Comment:
Why are we removing the style-css.js file? Isn't this always what is imported into the component in the `getStyles()` method?

Also, if we are moving away from that shouldn't we also get rid of `styles-fixed-css.js`?

But both of them are imported into the component.